scripts/compact-junit.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from junitparser import JUnitXml, Failure
+
+
+def main():
+    junit_xml_file = sys.argv[1]
+    suite = JUnitXml.fromfile(junit_xml_file)
+    for case in suite:
+        if not isinstance(case.result, Failure):
+            case.system_out = ''
+            case.system_err = ''
+    suite.write()
```

Comment:
All of the data is loaded into memory as objects, not xml. Therefore, xml metadata is lost during the read, which reduces file size. The xml metadata has no influence on the test results displayed.